app/src/main/java/com/yterletskyi/happyfriend/features/settings/domain/SettingEnum.kt

```diff
@@ -3,4 +3,5 @@ package com.yterletskyi.happyfriend.features.settings.domain
 enum class SettingEnum {
     MY_WISHLIST,
     APP_VERSION,
+    GENERAL_LIST,
```

Comment:
the name `GENERAL_LIST`is too broad and ambiguous, let's rename to `GENERAL_IDEAS`